misc_docs/syntax/decorator_uncurry.mdx

```diff
@@ -0,0 +1,34 @@
+---
+id: "uncurry-decorator"
+keywords: ["uncurry", "decorator"]
+name: "@uncurry"
+summary: "This is the `@uncurry` decorator."
+category: "decorators"
+---
+
+The `@uncurry` decorator allows you to annotate **external** function callback argument types
+so when writing the callbacks they may be declared as normal functions `() => { ... }`
+rather than needing the uncurried function syntax `(.) => { ... }`.
+
+### Example
+
```

Comment:
In the following example we are binding to a function `map(arr, callback)` and use the `@uncurry` annotation to make sure that `callback` is always treated as an uncurried function:

Reply:
Added 👍